server/src/test/java/org/apache/druid/server/coordinator/DataSourceCompactionConfigTest.java

```diff
@@ -69,6 +69,7 @@ public void testSerdeBasic() throws IOException
         null,
         null,
         null,
+        null,
```

Comment:
This class should have a couple of serde tests for both possible engine values.

Reply:
Modfieid a couple of existing tests to handle both engine types.